swan-lake/references/style-guide/coding-conventions/operators-keywords-and-types.md

```diff
@@ -54,7 +54,7 @@ string? name;
 **Do's**
 
 ```ballerina
-map<int|string> x;// map reference type
+map<int|string> x;// Map reference type.
```

Comment:
Here calling it a "reference type" is wrong. 
```suggestion
map<int|string> x;// Map built-in type.
```
or

```suggestion
map<int|string> x;// Map contraint type.
```
or simply,
```suggestion
map<int|string> x;// Map type.
```

Reply:
Since here we show, do's and don't on the way we write the constraint of the mapping type. `Map constraint type.` makes more sense to me. Hence, preferred.